### PR TITLE
JuliaFormatter config, reformat project

### PR DESCRIPTION
**Pull request by arlowhite**
Add format config file (same as ADRIA.jl)
Add Format workflow action.

### CODE REVIEW

File: src/site_assessment/geom_ops.jl [hunk 1]
```diff
@@ -165,9 +164,8 @@ function find_horizontal(geom::GIWrap.Polygon)::Vector{Tuple{Float64,Float64}}
     coords = collect(GI.coordinates(geom)...)
     first_coord = first(coords)
     second_coord = coords[
-        (getindex.(coords, 2).∈first_coord[2]) .&&
-        (getindex.(coords, 1).∉first_coord[1])
-    ]
+    (getindex.(coords, 2) .∈ first_coord[2]) .&& (getindex.(coords, 1) .∉ first_coord[1])
+]
```
Comment (arlowhite) [inline src/site_assessment/geom_ops.jl:168]: I like the old code better :(
seems like a JuliaFormatter bug that it removes indent on closing bracket.

Is there a better way to write this code? or I could disable formatting here

**Review by PeterBaker0 — APPROVED** [review]

File: src/criteria_assessment/query_thresholds.jl [hunk 2]
```diff
@@ -109,8 +109,8 @@ end
 
 Obtain the pixel positions of valid data.
 
-Intended for use in something like:
-https://developers.arcgis.com/javascript/latest/sample-code/layers-imagery-pixelvalues/
+Intended for use in applications similar to:
+`https://developers.arcgis.com/javascript/latest/sample-code/layers-imagery-pixelvalues/`
```
Comment (ConnectedSystems) [inline src/criteria_assessment/query_thresholds.jl:113]: This isn't an example URL for the user, so format this as an actual link please

File: src/criteria_assessment/query_thresholds.jl [hunk 3]
```diff
@@ -201,7 +201,7 @@ applied to a set of criteria.
 
 # Notes
 - Zero values indicate locations to mask **out**.
-- Ones indicate locations to **keep**.
+- One values indicate locations to **keep**.
```
Comment (ConnectedSystems) [inline src/criteria_assessment/query_thresholds.jl:204]: How about "Zeros" and "Ones"?

Removes the need to say "values"

e.g., "Zeros indicate locations..."

File: src/site_assessment/geom_ops.jl (hunk 1 above)
Comment (arlowhite) [reply src/site_assessment/geom_ops.jl:185]: Created issue https://github.com/domluna/JuliaFormatter.jl/issues/876

**Pull request merged**